MekHQ/src/mekhq/campaign/parts/MissingAvionics.java

```diff
@@ -53,6 +53,15 @@ public MissingAvionics(int tonnage, Campaign c) {
     
     @Override 
 	public int getBaseTime() {
+        if (campaign.getCampaignOptions().useAeroSystemHits()) {
+            //Test of proposed errata for repair times
+            Entity e = unit.getEntity();
+            if (e.hasETypeFlag(Entity.ETYPE_DROPSHIP) || e.hasETypeFlag(Entity.ETYPE_JUMPSHIP)) {
+                return 4800;
```

Comment:
One final note on the previous 2 inline comments' remarks about consistency: here the large craft is 8 times the small craft time. Major WTF factor ;)